tox.ini

```diff
@@ -90,11 +90,13 @@ commands =
     notudf: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} and not udf" {posargs:} src/snowflake/snowpark tests
     udf: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} src/snowflake/snowpark tests
     notdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} tests
+    multithreaded: {env:SNOWFLAKE_PYTEST_CMD} --multithreading_mode -m "{env:SNOWFLAKE_TEST_TYPE} and multithreaded_run" {posargs:} tests
     notudfdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} and not udf" {posargs:} tests
     local: {env:SNOWFLAKE_PYTEST_CMD} --local_testing_mode -m "integ or unit or mock" {posargs:} tests
     dailynotdoctest: {env:SNOWFLAKE_PYTEST_DAILY_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} tests
     # Snowpark pandas commands:
     snowparkpandasnotdoctest: {env:MODIN_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} {env:SNOW_1314507_WORKAROUND_RERUN_FLAGS} tests/unit/modin tests/integ/modin tests/integ/test_df_to_snowpark_pandas.py
+    snowparkpandasmultithreaded: {env:MODIN_PYTEST_CMD} --multithreading_mode --skip_sql_count_check --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE} and multithreaded_run" {posargs:} {env:SNOW_1314507_WORKAROUND_RERUN_FLAGS} tests/integ/modin tests/integ/test_df_to_snowpark_pandas.py
```

Comment:
here we are only running tests/integ/modin tests/integ/test_df_to_snowpark_pandas.py? is that intended? i see you updated a set of modin test there

Reply:
we are running both `tests/integ/modin` and `tests/integ/test_df_to_snowpark_pandas.py`. The list is same as regular moding tests